inst/templates/TopPerformerGraph.r

```diff
@@ -5,8 +5,17 @@ library(pictoralist)
 # Create Pie Chart (Top Performer)
 run <- function(recip, data, spek){
   color_set <- c(PT$DL_BLUE, PT$DL_LIGHT_BORDER, PT$DL_LIGHT_BORDER)
-  percentage <- "90%"
-  goal <- 17/20
+
+  print(spek.measure.comparator.value)
```

Comment:
The value for `spek.measure.comparator.value` will eventually be made available by a spekex function.  I wouldn't expect this to exist currently.  Suggest removing this line.